### ISSUE TITLE
Usability issues in neuron search (minor)

### ISSUE THREAD

**Issue by jaysnoddy** [srid/neuron#302]
[edited to clarify issue and remove my ill-considered suggested and temporary fix]

Two minor usability issues in ``neuron search`` 

1. Often I do not know if I want to edit a file until looking at the files in search window.
    - It seems more natural to decide what to do with files *after* seeing them.
    - As it is now, the editor often opens when I do not need it *or* I need to start another process to edit the file.
2. I cannot select and copy text in the ``neuron search`` window.
     - Often I want to copy text from one or more files.
      -  Selecting text is such a common feature, I sometimes remember this issue only after a few failed attempts to select text.
      - A few times I have wanted to select text include: 
        - copying YAML frontmatter to reuse it in a new file.
        - adding multiple filenames to the clipboard stack.
        - text to paste into a much larger latex file 

  I realize this may be a lesser priority than some other new features but creating a better search window in the terminal would be nice to have sometime.

**Comment by srid** [comment]
You can use `neuron query` to further process the output of `neuron search`:

![image](https://user-images.githubusercontent.com/3998/88488192-b1306800-cf59-11ea-9acc-ff8232d00357.png)

I suppose we can make it easier by adding a `neuron search --id` that outputs only the ID.

**Comment by jaysnoddy** [comment]
Stupid me--Somehow, I did not notice it was already spitting back path + filename.

Still would like--eventually--ability to launch the editor from search screen without copy/paste, but minor/not urgent. 
Also sometimes after "search -e", I realize I do not need to edit, but it still drops me in editor.

I have just starting using neuron, but really like it!

**Comment by RoyiAvital** [comment]
A nice search feature would be adding a Front Page matter for Search Words.
Something the search engine will look at but won't pollute the Tags list.
